### PR TITLE
Changed RightFFVolts to rightFFVolts

### PR DESCRIPTION
**Pull request by jyu-22**
Changed RightFFVolts to rightFFVolts

### CODE REVIEW

**Comment by ashanawaz** [comment]
Nice!

**Comment by ngupta-22** [comment]
Looks good!

**Comment by cgjyang** [comment]
Looks great!

File: 2021/Robot2021/src/main/cpp/subsystems/Drivetrain.cpp [hunk 1]
```diff
@@ -860,6 +860,7 @@ void Drivetrain::RamseteFollowerExecute(void)
 
 bool Drivetrain::RamseteFollowerIsFinished(void)
 {
+    m_trajTimer.Stop();
```
Comment (yabberyabber) [inline 2021/Robot2021/src/main/cpp/subsystems/Drivetrain.cpp:863]: Did you intend to add this line in this change?

Comment (jyu-22) [reply 2021/Robot2021/src/main/cpp/subsystems/Drivetrain.cpp:863]: I was trying to make this a separate pull request but I think I messed up somewhere along the process

**Review by yabberyabber — APPROVED** [review]
Looks good! Happy to let both commits merge at once

File: 2021/Robot2021/src/main/cpp/subsystems/Drivetrain.cpp (hunk 1 above)
Comment (jmullins16) [reply 2021/Robot2021/src/main/cpp/subsystems/Drivetrain.cpp:863]: By calling the Stop() method, the timer will actually stop. But when is RamseteFollowerIsFinished() called? Is it only called when the command actually ends?

**Pull request merged**